fe/fe-core/src/test/java/com/starrocks/analysis/StorageVolumeTest.java

```diff
@@ -0,0 +1,120 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.analysis;
+
+import com.starrocks.sql.analyzer.AnalyzeTestUtil;
+import com.starrocks.sql.ast.AlterStorageVolumeStmt;
+import com.starrocks.sql.ast.CreateStorageVolumeStmt;
+import com.starrocks.sql.ast.DescStorageVolumeStmt;
+import com.starrocks.sql.ast.DropStorageVolumeStmt;
+import com.starrocks.sql.ast.ShowStorageVolumesStmt;
+import com.starrocks.sql.ast.StatementBase;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageVolumeTest {
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        AnalyzeTestUtil.init();
+    }
+
+    @Test
+    public void testCreateStorageVolumeParserAndAnalyzer() {
+        String sql = "CREATE STORAGE VOLUME storage_volume_1 type = s3 (\"aws.s3.region\"=\"us-west-2\") " +
+                "STORAGE_LOCATIONS = ('s3://xxx', 's3://yyy')";
+        StatementBase stmt = AnalyzeTestUtil.analyzeSuccess(sql);
+        Assert.assertTrue(stmt instanceof CreateStorageVolumeStmt);
+        Assert.assertEquals("CREATE STORAGE VOLUME 'storage_volume_1' TYPE = s3 (\"aws.s3.region\"" +
+                        " = \"us-west-2\") STORAGE_LOCATIONS = ('s3://xxx', 's3://yyy') ENABLED = true",
```

Comment:
why not use `STORAGE LOCATIONS`

Reply:
We decided to change from `STORAGE LOCATIONS`-> `LOCATIONS`.